app/controllers/auth_sources_controller.rb

```diff
@@ -2,11 +2,15 @@ class AuthSourcesController < ApplicationController
   def index
     @auth_sources = AuthSource.except_hidden
     @users = User.except_hidden.includes(:auth_source)
-    @auth_source_ldaps = resource_base_search_and_page.only_ldap
+    @auth_source_ldaps = resource_base_search_and_page
   end
 
   private
 
+  def model_of_controller
+    AuthSourceLdap
```

Comment:
The controller should show other auth sources as well, this looks odd?

Reply:
No, it shouldn't, the other auth sources are not editable.
And the `@auth_sources` are not affected by this change, only the `@auth_sources_ldaps`